CONTRIBUTING.md

```diff
@@ -87,7 +87,7 @@ All code in oneDNN gets promoted to product branches (`master`, `rls-`, and
 To simplify the work of reviewers, make sure that the commits in the pull
 request adhere to the following requirements:
 
-- Commit message should be fit into 50 (at most 72) characters and have the
+- First line of commit message should be fit into 100 characters and have the
```

Comment:
(nit) I honestly have no opinion on this one, but I think the 50/72 rule is pretty standard.

Reply:
Allright, will drop this change for now.